src/__benchmarks__/benchmarks.ts

```diff
@@ -54,13 +54,21 @@ async function runBenchmarks() {
   const benchs = await Promise.all(
     Object.entries(benchmarks).map(
       async ([bench, { query, schema, variables }]) => {
-        const compiledQuery = compileQuery(schema, query, undefined);
+        const compiledQuery = compileQuery(schema, query, undefined, {
+          debug: true
+        } as any);
         if (!isCompiledQuery(compiledQuery)) {
-          // tslint:disable-next-line
+          // tslint:disable-next-line:no-console
           console.error(`${bench} failed to compile`);
           return null;
         }
-
+        // tslint:disable-next-line:no-console
+        console.log(
+          `size of function for ${bench}: ${
+            (compiledQuery as any)
+              .__DO_NOT_USE_THIS_OR_YOU_WILL_BE_FIRED_compilation.length
```

Comment:
this will not be realistic. We should remove white-space and measure this. If we align the code for better white-space, this number will increase or decrease unintended.